pandas/io/common.py

```diff
@@ -274,14 +274,15 @@ def get_filepath_or_buffer(filepath_or_buffer, encoding=None,
             import boto
         except:
             raise ImportError("boto is required to handle s3 files")
-        # Assuming AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
+        # Assuming AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and AWS_S3_HOST
```

Comment:
can we document this in some doc-strings and/or a separate section in the docs? (e.g. it may be too long for the actual doc-strings, so might be better to make a section on how to do S3 access and just point to it).